packages/account-v2/src/containers/PersonalDetailsFormWithExample/PersonalDetailsFormWithExample.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { lazy, Suspense, useMemo } from 'react';
+import { Field, FieldProps, useFormikContext } from 'formik';
+import { InferType } from 'yup';
+import { Checkbox, InlineMessage, Loader, Text } from '@deriv-com/ui';
+import { DatePicker } from '../../components/DatePicker';
+import { FormInputField } from '../../components/FormFields';
+import { getNameDOBValidationSchema } from '../../utils/personal-details-utils';
+
+const ExampleImage = lazy(() => import('../../assets/proof-of-identity/personal-details-example.svg'));
+
+type TPersonalDetailsFormWithExampleFormProps = InferType<ReturnType<typeof getNameDOBValidationSchema>>;
+
+export const PersonalDetailsFormWithExample = () => {
+    const formik = useFormikContext<TPersonalDetailsFormWithExampleFormProps>();
+
+    if (!formik) {
+        throw new Error('PersonalDetailsFormWithExample must be used within a Formik component');
+    }
+
+    const { errors, values } = formik;
+
+    const { dateOfBirth, firstName, lastName } = values;
+
+    const isDisabled = useMemo(() => {
+        return (
+            !firstName ||
+            !lastName ||
+            !dateOfBirth ||
+            !!errors?.firstName ||
+            !!errors?.lastName ||
+            !!errors?.dateOfBirth
+        );
+    }, [firstName, lastName, dateOfBirth, errors.firstName, errors.lastName, errors.dateOfBirth]);
+
+    return (
+        <section className='p-16 outline outline-1 outline-system-light-active-background lg:mx-24 rounded-default'>
+            <InlineMessage className='items-start mb-16' variant='warning'>
+                <Text as='p' className='text-sm lg:text-default'>
+                    To avoid delays, enter your <span className='font-bold'>name</span> and{' '}
```

Comment:
is this `{' '}` intentional?

Reply:
I think the formatting is adding that space. Its added by prettier